### ISSUE TITLE
No Pokestops showing up on my phone

### ISSUE THREAD

**Issue by iPetrov2** [finndev/PokeBuddy#1120]
is this a soft ban or a perm ban?

**Comment by umbananas** [comment]
probably just a soft ban, OR more likely bad connection. 
It happens to me all the time, just turn it off and on again.

**Comment by iPetrov2** [comment]
@umbananas ill wait till like 10pm to get back on. i got a question though. i put the bot in my local city like 10 minutes after i went to central park. could that be a reason the pokestops in my local city wont give  me xp?

**Comment by umbananas** [comment]
Yes. I believe they are detecting walk pattern now. I think you won't be able to login if it's a hard ban. 

Personally I would wait until Pokebuddy includes the "humanization" feature before I bot again.

**Comment by Grundy4432** [comment]
Or just bot on a 2nd account for fun. I mean, that is what I'm doing. I actually walk around and play my main. Then I have a second account to bot for fun.
